### PR TITLE
Feat more standards

### PR DESCRIPTION
**Pull request by linuxbandit**
Changes:

- Add restart policies for containers
- Improve matchingcondition for route for traefik

Also I added two entries in the menu, a placeholder for cachet and a link to the helpdesk (i think the more, the merrier...).
Finally I renamed "core" to something more plausible

### CODE REVIEW

**Comment by serge1peshcoff** [comment]
why merge commit though?

**Comment by linuxbandit** [comment]
good question, I think I committed but couldn't push unless this - can't recall now, I think the remote went ahead

**Comment by linuxbandit** [comment]
@serge1peshcoff merge when ok :) (I see this is not yet approved, but when you do then also merge pls)

File: client/menu.json [hunk 1]
```diff
@@ -6,11 +6,21 @@
...
         "label": "Dashboard",
         "icon": "fa-tachometer-alt"
+      },
+      {
+        "name": "oms.helpdesk",
+        "label": "Need help?",
+        "icon": "fa-headset"
+      },
+      {
+        "name": "oms.status",
```
[the remark this answers is not in the document]
Comment (linuxbandit) [reply client/menu.json]: it works without issues (doesn\t complain, just shows an empty page lol)

[the remark this answers is not in the document]
Comment (serge1peshcoff) [reply client/menu.json]: then can you remove it? no use for it atm

[the remark this answers is not in the document]
Comment (linuxbandit) [reply client/menu.json]: i will

**Pull request closed without merging**